pytest.sh

```diff
@@ -0,0 +1 @@
+docker exec -it planet-read_py-backend_1 pytest
```

Comment:
I'm fine with manually running this command rather than putting it into a script, since we already manually run backend linting.  Perhaps we ought to make a util script in a future pr

up to you tho!

Reply:
Yep, replaced it with a utils script. Mostly for selfish reasons; I can delete when I clean this up for review.